### PR TITLE
add suggestive fallback messages on unavailable fonts

### PR DESCRIPTION
**Pull request by tcrayford**
as per https://github.com/jwilm/alacritty/issues/39

### CODE REVIEW

**Comment by tcrayford** [comment]
screen shot 2017-01-08 at 9 34 58 pm

File: font/src/darwin/mod.rs [hunk 1]
```diff
@@ -110,7 +110,11 @@ impl ::std::fmt::Display for Error {
                 write!(f, "Glyph not found for char {:?}", c)
             },
             Error::MissingFont(ref desc) => {
-                write!(f, "Couldn't find a font with {}", desc)
+                if (*desc).name == "monospace" {
```
Comment (jwilm) [inline font/src/darwin/mod.rs]: Shouldn't need to dereference here

File: font/src/darwin/mod.rs [hunk 2]
```diff
@@ -110,7 +110,11 @@ impl ::std::fmt::Display for Error {
...
+                    write!(f, "Couldn't find a font with {}.\n\tIt looks like you might be using the default config file, maybe you need to edit the font family to Menlo or another available font in alacritty.yml", desc)
```
Comment (jwilm) [inline font/src/darwin/mod.rs]: Please keep lines to 100 columns or less

File: font/src/darwin/mod.rs [hunk 3]
```diff
@@ -110,7 +110,11 @@ impl ::std::fmt::Display for Error {
...
+                } else {
+                    write!(f, "Couldn't find a font with {}.\n\tYou'll need to edit alacritty.yml to use an available font.", desc)
```
Comment (jwilm) [inline font/src/darwin/mod.rs]: Same here

File: font/src/ft/mod.rs [hunk 4]
```diff
@@ -252,7 +252,7 @@ impl ::std::fmt::Display for Error {
                 err.fmt(f)
             },
             Error::MissingFont(ref desc) => {
-                write!(f, "Couldn't find a font with {}", desc)
+                write!(f, "Couldn't find a font with {}.\n\tYou'll need to edit alacritty.yml to use an available font.", desc)
```
Comment (jwilm) [inline font/src/ft/mod.rs]: And here

**Comment by tcrayford** [comment]
@jwilm Ok, done. I've done that as a new commit, but if you'd rather I squash the commits together, I can do that as well.

**Comment by jwilm** [comment]
Yeah squashing sounds right. Thanks!

**Comment by tcrayford** [comment]
@jwilm done

**Pull request merged**